src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -345,18 +350,33 @@ public RoutedCommand LeftClickCommand
 
         public bool IsResizable { get; protected set; }
 
+        private void UpdateSceneItems()
+        {
+            if (Model3DDictionary == null)
+            {
+                sceneItems = new List<Model3D>();
+                return;
+            }
+
+            var values = Model3DDictionary.Values.ToList();
+            if (Camera != null)
+            {
+                values.Sort(new Model3DComparer(Camera.Position));
+            }
```

Comment:
For some reasons `Camera` is null in tests when running in self service, after adding this null check tests passed successfully:
![image](https://cloud.githubusercontent.com/assets/7658189/14277096/49f7f638-fb2a-11e5-9332-6b359bb06234.png)